lib/config/recommended.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
     'require-iframe-title': 'error',
     'require-input-label': 'error',
     'require-lang-attribute': 'error',
+    'require-presentational-children': 'error',
```

Comment:
```suggestion
    'require-presentational-children': 'warn',
```
Let's change this to warn so it won't break people. 

Also please add this to the lib/config/a11y.js:

```
'require-presentational-children': 'error',
```

Reply:
👌

Reply:
Even though this won't break peoples lint build, this doesn't seem like a common thing to do for new rules. Personally I would prefer not to have new warnings pop up when updating ember-template-lint to the next minor.